geo/src/lib.rs

```diff
@@ -28,6 +28,19 @@ pub mod algorithm;
 #[macro_use]
 extern crate approx;
 
+// Mean radius of Earth in meters
```

Comment:
I'd add a note (or maybe just link to the explanations in #176), or something that makes the distinction between uses of mean spherical and ellipsoidal earth radii here somewhere, just so we don't have to remember which is used for what application when this comes up again.